Apps/phexpanse/expanse.json

```diff
@@ -0,0 +1,533 @@
+                },
+                {
+                    "data_path": "action_result.data.0.data.*.lastObserved",
+                    "data_type": "string",
+                    "contains": [],
+                    "column_name": "Last Seen",
+                    "column_order": 4
+                },
+                {
+                    "data_path": "action_result.data.0.data.*.lastSampledIp",
+                    "data_type": "string",
+                    "contains": [
+                        "ip"
+                    ],
+                    "column_name": "Most Recent IP",
+                    "column_order": 5
+                },
+                {
+                    "data_path": "action_result.data.0.data.*.providers.0.name",
+                    "data_type": "string",
+                    "contains": [],
+                    "column_name": "Provider",
+                    "column_order": 6
+                },
+                {
+                    "data_path": "action_result.data.0.data.*.whois.0.registryExpiryDate",
+                    "data_type": "string",
+                    "contains": [],
+                    "column_name": "Expiration Date",
+                    "column_order": 7
+                },
+                {
+                    "data_path": "action_result.data.0.data.*.whois.0.nameServers",
+                    "data_type": "string",
+                    "contains": [
+                        "domain"
+                    ],
+                    "column_name": "Name Servers",
+                    "column_order": 8
+                },
+                {
+                    "data_path": "action_result.data.0.data.*.whois.0.registrar.name",
+                    "data_type": "string",
+                    "contains": [],
+                    "column_name": "Registrar",
+                    "column_order": 9
+                },
+                {
+                    "data_path": "action_result.data.0.data.*.whois.0.registrant.emailAddress",
+                    "data_type": "string",
+                    "contains": [
+                        "email"
+                    ],
+                    "column_name": "Registrant",
+                    "column_order": 10
+                }
+            ],
+            "render": {
+                "type": "table"
+            },
+            "versions": "EQ(*)"
+        },
+        {
+            "action": "lookup certificate",
+            "identifier": "lookup_certificate",
+            "description": "Returns certificates from Expanse for a full or partial common name.",
+            "verbose": "",
+            "type": "investigate",
+            "read_only": false,
+            "parameters": {
+                "common_name": {
+                    "description": "Search for given certificate value via a domain substring match.",
+                    "data_type": "string",
+                    "required": true,
+                    "primary": true,
```

Comment:
Primary is false because there are no values in contains. Remove contains key since empty. Remove value_list and default, too